components/common/src/main/java/com/hotels/styx/common/logging/HttpRequestMessageLogger.java

```diff
@@ -36,11 +35,23 @@ public HttpRequestMessageLogger(String name, boolean longFormatEnabled) {
     }
 
     public void logRequest(HttpRequest request, Origin origin) {
-        logger.info("requestId={}, request={}", request.id(), information(request, origin, longFormatEnabled));
+        if (request == null) {
+            logger.warn("Unable to log null request, origin={}", origin);
+        } else {
+            logger.info("requestId={}, request={}", request.id(), information(request, origin, longFormatEnabled));
+        }
     }
 
     public void logResponse(HttpRequest request, HttpResponse response) {
-            logger.info("requestId={}, response={}", request.id(), information(response, longFormatEnabled));
+        if (response == null) {
+            logger.warn("Unable to log null response, request={}", request);
+        } else {
+            logger.info("requestId={}, response={}", id(request), information(response, longFormatEnabled));
```

Comment:
The two messages should be somewhat consistent. Now we have:

> "Unable to log null response, request={...}"

vs

> "requestId={...}, response={...}, requestId, ...."

For consistency, when response==null, we would instead log:

> "requestId=NA, respone={...}"

But if it is possible for request to be `null`, it does beg a question can request also be sometimes `null`?